net/net.sh

```diff
@@ -840,6 +840,32 @@ stop() {
   echo "Stopping nodes took $SECONDS seconds"
 }
 
+
+checkPremptibleInstances() {
+  # The fullnodeIpList nodes may be preemptible instances that can disappear at
+  # any time.  Try to detect when a fullnode has been preempted to help the user
+  # out.
+  #
+  # Of course this isn't airtight as an instance could always disappear
+  # immediately after its successfully pinged.
+  for ipAddress in "${fullnodeIpList[@]}"; do
+    (
+      set -x
+      ping -o -t 4 "$ipAddress"
+    ) || {
+      cat <<EOF
+
+Warning: $ipAddress may have been preempted.
+
+Run |./gce.sh config| to restart it
+EOF
+      exit 1
+    }
+  done
+}
+
+checkPremptibleInstances
```

Comment:
This only needs to run in the `{re,}start` code paths, right?

Reply:
All commands here interact with one or more instances so I figured just do it all the time for now (except for when `--no-deploy` is given but not sure that's worth making the implementation much more complicated -- let's see if anybody complains)